prometheus-stats

```diff
@@ -44,6 +45,19 @@ def main():
     db_conn = sqlite3.connect(opts.db)
     cursor = db_conn.cursor()
 
+    if opts.prune:
+        # this would be much more elegant with just "PRAGMA foreign_keys = ON", and letting SQLite
+        # auto-cleanup deleted foreign keys in the other tables; but that takes unbearably long
```

Comment:
FTR: I let that run locally for about 15 minutes until I gave up. So I have no idea whether we are talking hours or even days here -- it takes a long time to do that existence check for all of the 9643613 (!) entries of the `Tests` table.

Reply:
:exploding_head: